cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -176,6 +180,7 @@ void WholeStageResultIterator::setConfToQueryContext(const std::shared_ptr<velox
   }
   // To align with Spark's behavior, set casting to int to be truncating.
   configs[velox::core::QueryConfig::kCastIntByTruncate] = std::to_string(true);
+  configs[velox::core::QueryConfig::kSpillEnabled] = std::to_string(true);
```

Comment:
It's noticed there are some other configs controlling Velox spill. Are these needed? Eg., `aggregationSpillEnabled`, `joinSpillEnabled`, `maxSpillFileSize` etc.

Reply:
`xxxSpillEnabled` are defaulted to true.

For the others (e.g. maxSpillFileSize), maybe we can tuning against them later on.